venues/ICML.cc/2017/MLAV/python/submission_done.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/python
```

Comment:
Well, in UAI I called create_invitations.py. That's was the final goals besides you need to create a bunch of groups to setup the invitation.  But we can find another more descriptive name.